package.json

```diff
@@ -161,6 +161,11 @@
             }
         ],
         "commands": [
+            {
+                "command": "tsp.openConfigWebview",
+                "title": "Open Config Webview",
```

Comment:
```suggestion
                "title": "Edit TSP Configuration",
```

Reply:
Also, when I debug the extension I see this: 
![image](https://github.com/user-attachments/assets/0be9b3c9-5473-44ca-bb6a-c4bd5d03b22a)